execution/executor/src/lib.rs

```diff
@@ -51,18 +54,21 @@ static OP_COUNTERS: Lazy<libra_metrics::OpMetrics> =
 /// `Executor` implements all functionalities the execution module needs to provide.
 pub struct Executor<V> {
     rt: Runtime,
-
     /// Client to storage service.
     storage_read_client: Arc<dyn StorageRead>,
     storage_write_client: Arc<dyn StorageWrite>,
-
+    cache: SpeculationCache,
     phantom: PhantomData<V>,
 }
 
 impl<V> Executor<V>
 where
     V: VMExecutor,
```

Comment:
Do we have other VMExecutor? Yes, we do... Is it useful?

Reply:
There is a mock one.